flags.ts

```diff
@@ -25,4 +26,20 @@ export const DEFAULT_FLAGS = [
   '--mute-audio',
   // Skip first run wizards
   '--no-first-run',
+
+  // Disable timers being throttled in background pages/tabs
+  '--disable-background-timer-throttling',
```

Comment:
does this have any benefit for us? It seems like it could have an unrealistic effect on performance in certain corner cases of sites and it may be unnecessary for Lighthouse due to always being the foreground tab

Reply:
> unnecessary for Lighthouse due to always being the foreground tab

yes seems correct.

however since other projects use chrome-launcher... this would affect anyone attempting to use a pool of tabs to do a bunch of work.